OfficeApp3Web/Scripts/InteractiveSDKv2.js

```diff
@@ -71,15 +71,33 @@ InteractiveTutorial.App = new function () {
         */
         $("body").keydown(function (event) {
             if (event.which == 27) {
-                $("#run").focus();
+                if ($("#headercontent")[0].getAttribute("class").toString() == "apiPageHeader") {
+                    $("#run").focus();
+                }
+                if ($("#codeSnippet")[0].getAttribute("class").toString() != "hidden") {
+                    $("#runButton").focus();
+                }
                 $('#toastMessage').hide();
             } //Check for enter and space to mimic the behavior of buttons
             else if (event.which == 13 || event.which == 32) {
                 var element = $(event.srcElement);
                 if (element.attr("role") == "button") {
                     $(event.srcElement).click();
+                    return false;
```

Comment:
I have checked that no logic need update, because the logic to disable the origin function of enter or space key only be applied on some special cases, for example, when the key down event is triggered by a button. So the logic have no influence on the function of these keys in the code editor area.